Marlin/src/lcd/extensible_ui/ui_api.h

```diff
@@ -49,161 +49,211 @@ typedef const __FlashStringHelper *progmem_str;
+  void        stopPrint                      ();
+  void        pausePrint                     ();
+  void        resumePrint                    ();
 
   class FileList {
     private:
       uint16_t num_files;
 
     public:
-      FileList();
-      void        refresh();
-      bool        seek(uint16_t, bool skip_range_check = false);
-
-      const char *longFilename();
-      const char *shortFilename();
-      const char *filename();
-      bool        isDir();
-
-      void        changeDir(const char *dirname);
-      void        upDir();
-      bool        isAtRootDir();
-      uint16_t    count();
+      FileList                               ();
+      void        refresh                    ();
+      bool        seek                       (uint16_t, bool skip_range_check = false);
+
+      const char *longFilename               ();
+      const char *shortFilename              ();
+      const char *filename                   ();
+      bool        isDir                      ();
+
+      void        changeDir                  (const char *dirname);
+      void        upDir                      ();
+      bool        isAtRootDir                ();
+      uint16_t    count                      ();
   };
 
-  // The following event handlers are to be declared by the extension
-  // module and will be called by Marlin.
-
-  void onStartup();
-  void onIdle();
-  void onMediaInserted();
-  void onMediaError();
-  void onMediaRemoved();
-  void onPlayTone(const uint16_t frequency, const uint16_t duration);
-  void onPrinterKilled(const char* msg);
-  void onPrintTimerStarted();
-  void onPrintTimerPaused();
-  void onPrintTimerStopped();
-  void onFilamentRunout();
-  void onStatusChanged(const char* msg);
-  void onStatusChanged(progmem_str msg);
-  void onFactoryReset();
-  void onStoreSettings();
-  void onLoadSettings();
+  /* Event callback routines */
+  /* Should be declared by EXTENSIBLE_UI and will be called by Marlin */
+
+  void        onStartup                      ();
+  void        onIdle                         ();
+  void        onMediaInserted                ();
+  void        onMediaError                   ();
+  void        onMediaRemoved                 ();
+  void        onPlayTone                     (const uint16_t frequency, const uint16_t duration);
+  void        onPrinterKilled                (const char* msg);
+  void        onPrintTimerStarted            ();
+  void        onPrintTimerPaused             ();
+  void        onPrintTimerStopped            ();
+  void        onFilamentRunout               ();
+  void        onStatusChanged                (const char* msg);
+  void        onStatusChanged                (progmem_str msg);
+  void        onFactoryReset                 ();
+  void        onStoreSettings                ();
+  void        onLoadSettings                 ();
 };
+
+/* Helper macros to increment or decrement a value. For example:
```

Comment:
For consistency and for the sake of code-folding, please format block comments like so…
```cpp
/**
 * First line of comment
 *
 * More stuff
 */
```

Reply:
Some editors will distinguish between ordinary comments and Doxygen/HeaderDoc style comments.

Reply:
Some editors will distinguish between ordinary comments and Doxygen/HeaderDoc style comments.